src/components/cart/OrderSummary/OrderSummary.tsx

```diff
@@ -0,0 +1,44 @@
+import type { InputHTMLAttributes, ReactNode } from 'react'
+import React from 'react'
+import { useFormattedPrice } from 'src/sdk/product/useFormattedPrice'
+
+import './order-summary.scss'
+
+export interface OrderSummaryProps extends InputHTMLAttributes<HTMLDivElement> {
```

Comment:
This interface is extending from `InputHTMLAttributes<HTMLDivElement>`, but none of the other possible props are really being used by the component, so I suggest we keep it simple:

```suggestion
export interface OrderSummaryProps {
```